packs/erlang/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM erlang:20.3.6-alpine as builder
+
+RUN apk add --update tar curl git bash make libc-dev gcc g++ && \
+    rm -rf /var/cache/apk/*
+
+RUN set -xe \
+    && curl -fSL -o rebar3 "https://s3.amazonaws.com/rebar3-nightly/rebar3" \
```

Comment:
hey @tsloughter - what is rebar3? is this an erlang package? are erlang packages normally stored in s3?

Reply:
Looks like a build tool for Erlang apps

Would it be better to use a stable release instead of the nightlies? https://github.com/erlang/rebar3/releases

Reply:
Thanks, good catch. I've updated it to use the tagged release. I upload those to `s3://rebar3/rebar3` after publishing them on github, so that s3 link is always the latest stable release.

Reply:
awesome thank you